### ISSUE TITLE
Valid support operation with long 64-bit java

### ISSUE THREAD

**Issue by nicolaichuk** [dragome/dragome-sdk#161]
copy from https://github.com/dragome/dragome-sdk/issues/157

javascript work only with 32-bit value when used bitwise operation.

https://stackoverflow.com/questions/3081271/number-of-bits-to-represent-a-number

And used 53-bit value for other integer operation

https://stackoverflow.com/questions/17320706/javascript-long-integer

I do not sure about valid mirror operation with 64-bit long value between java and javascript code.
Maybe need write same wrapper over java native long value.

**Comment by fpetrola** [comment]
Yes, we can use something like https://github.com/peterolson/BigInteger.js, but it's not a small change for the compiler.
